extensions/sockets/src/main/java/org/mule/module/socket/api/exceptions/UnresolvableHostException.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.socket.api.exceptions;
+
+public class UnresolvableHostException extends RuntimeException
+{
+
+    public UnresolvableHostException(String message)
```

Comment:
the exception exists yet I don't see the validate methods catching it so that they can put the proper FaultCode on a ConnectionException